packages/types/custom/core.ts

```diff
@@ -531,11 +531,13 @@ export interface PolarError {
 
 export interface CoreState {
   map: number
+  center: [number, number] | null
   clientHeight: number
   clientWidth: number
   components: number
   configuration: MapConfig
   errors: PolarError[]
+  plugin: object
```

Comment:
I can't seem to find where this resides. I can find `modules.plugin`, but that is not what is typed here

Reply:
It's e.g. used in `packages/clients/meldemichel/src/store/module.ts`:
`let address = rootState?.plugin?.addressSearch?.chosenAddress`

Reply:
Weird, that I can't seem to find the place in the core 🎩